docs: Add frontmatter to `reference/*` pages (#23093)
MystenLabs/sui: docs/content/references/sui-api.mdx

---
title: Sui RPC

> description: SuiJSON is a JSON-based format that enables JSON inputs to be aligned with Move call arguments. 

Reviewer: ```suggestion
description: SuiJSON is a JSON-based format that enables alignment of JSON inputs with Move call arguments. 
```
